pyproject.toml

```diff
@@ -146,6 +146,7 @@ cibuildwheel = "2.10.0"
 editorconfig-checker = "2.4.0"
 types-requests = "^2.28"
 boto3-stubs = "^1.26"
+patchelf = { version="^0.17.2.0", markers="platform_system=='Linux'" }
```

Comment:
`patchelf` is used during the build step, you need to also specify it's version here (in `build-system -> requires`)

Reply:
done in 9894197